### PR TITLE
Feat/add methods to root of numbers

### PR DESCRIPTION
**Pull request by Dovchik**
Add available and active numbers to root of Numbers client.
Leaving added methods as obsolete in own tag for backward compatibility.

### CODE REVIEW

**Review by asein-sinch — COMMENTED** [review]
Can you also update the tests to reflect the deprecation of the `Available` and `Active` domains?

File: src/Sinch/Numbers/Active.cs [hunk 1]
```diff
@@ -91,7 +101,9 @@ public ActiveNumbers(string projectId, Uri baseAddress,
             _http = http;
         }
 
-        public Task<ListActiveNumbersResponse> List(ListActiveNumbersRequest request, CancellationToken cancellationToken = default)
+
+        public Task<ListActiveNumbersResponse> List(ListActiveNumbersRequest request,
```
Comment (asein-sinch) [inline src/Sinch/Numbers/Active.cs:105]: I'm curious about why the formatting changed as there is no configuration change in this PR. Is it linked to the IDE? I just want to make sure this is intentional and won't be reverted by an auto-formatting in another PR.

File: src/Sinch/Numbers/Numbers.cs [hunk 2]
```diff
@@ -25,6 +35,42 @@ public interface ISinchNumbers
         ///     release numbers from projects, or list all numbers assigned to a project.
         /// </summary>
         public ISinchNumbersActive Active { get; }
```
Comment (asein-sinch) [inline src/Sinch/Numbers/Numbers.cs:37]: You may also mark this as obsolete before changing the visibility in the next major

File: src/Sinch/Numbers/Active.cs (hunk 1 above)
Comment (Dovchik) [reply src/Sinch/Numbers/Active.cs:105]: Auto formatted by IDE agnostic `dotnet format`

**Review by asein-sinch — APPROVED** [review]

**Pull request merged**